Improve default template node titles. No recent signers on share pages.

CharlieSeagull opened the pull request:
- add description to MP data groupset
- do not show recent signer block on share pages
- add "do not delete" to default templates
- add fields to supporter.php

torotil commented on modules/campaignion_supporter/src/Supporter.php:
```diff
@@ -136,6 +136,8 @@ public function exporter($type, $language) {
         $map['CAMPAIGN'] = new TagField('campaign_tag');
         $map['TAGS'] = new TagsField('supporter_tags', TRUE);
+        $map['ONETIMEVAL'] = new WrapperField('field_onetimeval');
+        $map['REGVAL'] = new WrapperField('field_regval');
```
The fields don’t seem to exist in the starterkit. THere is no point in adding exporters for them if they don’t exist.

torotil commented on modules/campaignion_donation_type/campaignion_donation_type.features.uuid_node.inc:
```diff
@@ -12,7 +12,7 @@ function campaignion_donation_type_uuid_features_default_content() {
 
   $nodes[] = array(
-  'title' => 'Default Donation Form',
+  'title' => 'Default Donation Form - DO NOT DELETE',
```
Maybe also changing it to „Donation form **template** - DO NOT DELETE” would be still clearer.

torotil replied:
I woul also remove the „Default …” in the beginning then.

torotil replied:
Was there a discussion about changing the node titles anywhere? I think this is something we should get @dayjess and/or @clouised28 involved in.

CharlieSeagull replied:
Yes, the discussion was started by Jess. See link to mattermost conversation in ticket.

CharlieSeagull replied:
I would also leave the "Default" in the title, for the reason that there can be many templates, but this is the original, "default" one.

CharlieSeagull committed:
Small improvements in starterkit

- do not show recent signer block on share pages
- add "do not delete" to default templates
